src/GitLink/Extensions/PdbExtensions.cs

```diff
@@ -69,28 +58,19 @@ public static class PdbExtensions
             var results = new List<Tuple<string, byte[]>>();
             foreach (var value in values)
             {
-                if (!value.Name.Contains(FileIndicator))
+                if (!value.Name.StartsWith(FileIndicator))
                 {
                     continue;
                 }
 
                 var num = value.Stream;
                 var name = value.Name.Substring(FileIndicator.Length);
 
-                var bytes = pdbFile.ReadStreamBytes(num);
-                if (bytes.Length != 88)
-                {
-                    continue;
-                }
-
                 // Get last 16 bytes for checksum
-                var buffer = new byte[16];
-                for (int i = 72; i < 88; i++)
-                {
-                    buffer[i-72] = bytes[i];
-                }
-
-                results.Add(new Tuple<string, byte[]>(name, buffer));
+                var bytes = pdbFile.ReadStreamBytes(num);
+                var checksum = new byte[16];
```

Comment:
Actually in a later iteration I change this more. At this iteration (I'd forgotten this when I wrote the description for this PR), I didn't understand what made the streams different lengths so I just took the last 16 bytes. But later, I realize it is the checksum length that makes the difference in stream length. So sit tight. It gets better. :)